pom.xml

```diff
@@ -652,7 +652,7 @@
                 <module>apache-httpclient</module>
                 <module>apache-kafka-2</module>
                 <module>apache-kafka-3</module>
-                <module>apache-kafka</module>
+                <!--<module>apache-kafka</module>--><!--build issue in JAVA-30182-->
```

Comment:
Is this still relevant? If yes can we fix it here?

Reply:
Reverted